ProjectFiles/src/timelineManager/controller/MainWindowController.java

```diff
@@ -35,21 +35,39 @@
     @FXML
     private JFXButton goRightButton;
     
+    
+    
      //date of the day
-    private LocalDate currentDate = LocalDate.now();
+    public LocalDate currentDate = LocalDate.now();
```

Comment:
please make this field private again and send the field into the methods that needs it instead